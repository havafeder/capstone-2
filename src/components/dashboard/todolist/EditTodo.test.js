import React from "react";
import { render } from "@testing-library/react";
import EditTodo from "./EditTodo";
import { MemoryRouter } from "react-router";

it("matches snapshot", function () {
	const { asFragment } = render(
		<MemoryRouter>
			<EditTodo todo='Be kind'/>
		</MemoryRouter>,
	);
	expect(asFragment()).toMatchSnapshot();
});