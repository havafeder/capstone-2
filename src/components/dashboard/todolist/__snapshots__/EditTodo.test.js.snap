// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <button
    class="btn edit"
    data-target="#idundefined"
    data-toggle="modal"
    type="button"
  >
    edit
  </button>
  <div
    class="modal"
    id="idundefined"
  >
    <div
      class="modal-dialog"
    >
      <div
        class="modal-content"
      >
        <div
          class="modal-header"
        >
          <h4
            class="modal-title"
          >
            edit todo
          </h4>
          <button
            class="close"
            data-dismiss="modal"
            type="button"
          >
            Ã—
          </button>
        </div>
        <div
          class="modal-body"
        >
          <input
            class="form-control"
            type="text"
            value=""
          />
        </div>
        <div
          class="modal-footer"
        >
          <button
            class="btn btn-warning"
            data-dismiss="modal"
            type="button"
          >
            edit
          </button>
        </div>
        <div
          class="modal-footer"
        >
          <button
            class="btn btn-danger"
            data-dismiss="modal"
            type="button"
          >
            close
          </button>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
